queries.sql

```diff
@@ -140,20 +145,182 @@ vet-clinic=# SELECT * FROM animals WHERE neutered = true;
 
 --Find all animals not named Gabumon.
 
-vet-clinic=# SELECT * FROM animals WHERE NOT name = 'Gabumon';
+SELECT * FROM animals WHERE NOT name = 'Gabumon';
 -- id |  name   | date_of_birth | escape_attempts | neutered | weight_kg 
 ----+---------+---------------+-----------------+----------+-----------
 --  2 | Agumon  | 2020-02-03    |               0 | t        |     10.23
 --  4 | Pikachu | 2021-01-07    |               1 | f        |     15.04
 --  5 | Devimon | 2017-05-12    |               5 | t        |        11
 --(3 rows)
```

Comment:
- [ ] Please it would be really great you take to remove all comments from your queries.sql for a clean, readable and understandable code which is often considered a good practice, what do you think? 🤔